src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx

```diff
@@ -569,7 +600,7 @@ export function getSelectedPodList(selectedOption: string): PodMetaData[] {
                 handleDefaultForSelectedOption(selectedOption.replace('All new ', ''));
             } else if (selectedOption.startsWith('All old ')) {
                 handleDefaultForSelectedOption(selectedOption.replace('All old ', ''));
-            } else if (selectedOption.startsWith('All')) {
+            } else if (selectedOption.startsWith('All ')) {
```

Comment:
**Note:** This is added to handle one corner case where the user creates an app starting with 'all'. **E.g.** 'all-new-chart' or something similar.